src/utils.ts

```diff
@@ -129,9 +128,9 @@ export function getNpmConfig(runnerConfig: NpmConfigContainer) {
   if (runnerConfig.npm === undefined) {
     return {};
   }
-  const cfg: { [key: string]: string | boolean | null } = {
+  const cfg: { [key: string]: string | boolean | null | undefined } = {
     registry: runnerConfig.npm.registry || getDefaultRegistry(),
-    'strict-ssl': runnerConfig.npm.strictSSL !== false,
+    'strict-ssl': runnerConfig.npm.strictSSL,
```

Comment:
To use `'strict-ssl': runnerConfig.npm.strictSSL` directly, it needs to accept `string | boolean | null | undefined`. Changing the default to `undefined` is acceptable because `undefined` values are filtered out before setting the config, as seen [here](https://github.com/saucelabs/sauce-runner-utils/blob/main/src/npm.ts#L69).

Reply:
Speaking in general terms, I think that `boolean | null | undefined` are all acceptable values.

To elaborate:
- **boolean**: Self explanatory.
- **null** & **undefined**: Depends on how saucectl serializes it when the value is "not set". Is it explicit? Then the value would be `null`, which is what it does today. If `omitempty` were configured, then we'd end up with an `undefined` value in Javascript. I think it doesn't hurt to keep that flexibility here.

Reply:
Yeah good point. Now the unset value is `null` as `omitempty` is not set in `StrictSSL` json tag. So just keep it as it is now?

Reply:
Works for me 👍

Reply:
Besides, it's based on this discussion https://github.com/saucelabs/sauce-runner-utils/pull/76#discussion_r1718644021